extensions/amp-accordion/1.0/example/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <script src="https://cdn.ampproject.org/bento.js"></script>
+    <script
+      async
+      src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"
+    ></script>
+    <style>
```

Comment:
I think from our discussions yesterday, it'd be recommended to use a <link rel> to our `bento-accordion-1.0.css` instead of copy/pasting these styles

Reply:
How does including css via link solve the problem of styles being injected automatically?

Reply:
oh it doesn't, there's a separate PR that should stop injecting styles: https://github.com/ampproject/amphtml/pull/36873

I was suggesting using a <link> because that's probably easier to consumers than asking them top copy/paste our css